pax-cdi-extension/src/main/java/org/ops4j/pax/cdi/extension/impl/client/AbstractServiceInvocationHandler.java

```diff
@@ -12,29 +12,26 @@
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.ServiceReference;
 
-
 public abstract class AbstractServiceInvocationHandler<S> implements InvocationHandler {
-
-
-    protected InjectionPoint ip;
-    protected BundleContext bundleContext;
-    protected ServiceReference<S> serviceRef;
-    protected CdiContainer cdiContainer;
-    protected ServiceObjectsWrapper<S> serviceObjects;
-    protected S service;
-
-    @SuppressWarnings("unchecked")
-    protected AbstractServiceInvocationHandler(InjectionPoint ip) {
-        this.ip = ip;
-        this.bundleContext = InjectionPointOsgiUtils.getBundleContext(ip);
-        this.serviceRef = InjectionPointOsgiUtils.getServiceReference(ip);
-        this.serviceObjects = PrototypeScopeUtils.createServiceObjectsWrapper(bundleContext, serviceRef);
-        ServiceReference<CdiContainerFactory> serviceReference = bundleContext
-            .getServiceReference(CdiContainerFactory.class);
-        CdiContainerFactory cdiContainerFactory = bundleContext.getService(serviceReference);
-        this.cdiContainer = cdiContainerFactory.getContainer(bundleContext.getBundle());
-    }
-
-    public abstract void release();
-
+	
+	protected InjectionPoint			ip;
```

Comment:
Sorry for the whitespaces. I had my Eclipse autoformat turned on. Are these [coding styles](https://ops4j1.jira.com/wiki/display/ops4j/Coding+Style) still valid?